nc-news-fe/api.js

```diff
@@ -1,11 +1,21 @@
 import axios from 'axios'
 
-const getAllArticles = () => {
+export const getAllArticles = () => {
     return axios.get("https://nc-news-a8e3.onrender.com/api/articles")
         .then(res => res.data.articles)
         .catch(error => {
             console.error('Error fetching data:', error);
+            throw error; // Re-throw the error to handle it where the function is called
         });
 };
 
-export default getAllArticles;
\ No newline at end of file
+export const getArticleById = (article_id) => {
+    return axios.get(`https://nc-news-a8e3.onrender.com/api/articles/${article_id}`)

```

Comment:
In an effort to make this a little dryer, have a look at the axios notes for a custom instance. Your base url is not changing each time.